File: Users/Mery Zúñiga/Videos/Ingenieria3-master/Ingenieria3-master/SistemaDecomisos/src/java/model/ActaDestruccion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

/**
 *
 * @author Mery Zúñiga
 */
public class ActaDestruccion implements Serializable,Jsonable{
    private int idDestruccion;
    private Date fecha;
  //  private Policia policia;
    private Testigo testigo1;
    private Testigo testigo2;
    private Lugar lugar;
    private String encargado;
    private ActaDecomiso decomiso;

    public ActaDestruccion() {
    }

    public ActaDestruccion(int idDestrucion, Date fecha, /*Policia policia,*/ Testigo testigo1, Testigo testigo2, Lugar lugar, String encargado, ActaDecomiso decomiso) {
        this.idDestruccion = idDestrucion;
        this.fecha = fecha;
        //this.policia = policia;
        this.testigo1 = testigo1;
        this.testigo2 = testigo2;
        this.lugar = lugar;
        this.encargado = encargado;
        this.decomiso = decomiso;
    }

    public int getIdDestruccion() {
        return idDestruccion;
    }

    public Date getFecha() {
        return fecha;
    }

//    public Policia getPolicia() {
//        return policia;
//    }

    public Testigo getTestigo1() {
        return testigo1;
    }

    public Testigo getTestigo2() {
        return testigo2;
    }

    public Lugar getLugar() {
        return lugar;
    }

    public String getEncargado() {
        return encargado;
    }

    public ActaDecomiso getActaDecomiso() {
        return decomiso;
    }

    public void setIdDestruccion(int idDestrucion) {
        this.idDestruccion = idDestrucion;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

//    public void setPolicia(Policia policia) {
//        this.policia = policia;
//    }

    public void setTestigo1(Testigo testigo1) {
        this.testigo1 = testigo1;
    }

    public void setTestigo2(Testigo testigo2) {
        this.testigo2 = testigo2;
    }

    public void setLugar(Lugar lugar) {
        this.lugar = lugar;
    }

    public void setEncargado(String encargado) {
        this.encargado = encargado;
    }

    public void setActaDecomiso(ActaDecomiso actaDecomiso) {
        this.decomiso = actaDecomiso;
    }
    
    
}

File: Users/Mery Zúñiga/Videos/Ingenieria3-master/Ingenieria3-master/SistemaDecomisos/web/js/objects/ActaDestruccion.js
function ActaDestruccion(idDestruccion, fecha, /*policia,*/ testigo1, testigo2, lugar, encargado, decomiso) {
    this.ActaDestruccion(idDestruccion, fecha, /*policia,*/ testigo1, testigo2, lugar, encargado, decomiso);
}

ActaDestruccion.prototype = {
    idDestruccion: 0,
    fecha: "",
  //  policia: "",
    testigo1: "",
    testigo2: "",
    lugar: "",
    encargado: "",
    decomiso: "",
    ActaDestruccion: function (idDestruccion, fecha, /*policia,*/ testigo1, testigo2, lugar, encargado, decomiso) {
        this.idDestruccion = idDestruccion;
        this.fecha = fecha;
       // this.policia = policia;
        this.testigo1 = testigo1;
        this.testigo2 = testigo2;
        this.lugar = lugar;
        this.encargado = encargado;
        this.decomiso = decomiso;
    }
};

ActaDestruccion.from = function (plain) {
    acta = new ActaDestruccion(
            plain.idDestruccion,
            plain.fecha,
         //   plain.policia,
            plain.testigo1,
            plain.testigo2,
            plain.lugar,
            plain.encargado,
            plain.decomiso);
    return acta;
};

ActaDestruccion.to = function (acta) {
    return {
        _class: 'ActaDestruccion',
        idDestruccion: acta.idDestruccion,
        fecha: acta.fecha,
    //    policia: acta.policia,
        testigo1: acta.testigo1,
        testigo2: acta.testigo2,
        lugar: acta.lugar,
        encargado: acta.encargado,
        decomiso: acta.decomiso
    };
};

